src/Admin/BaseFieldDescription.php

```diff
@@ -292,7 +292,7 @@ public function getFieldValue($object, $fieldName)
         }
 
         foreach ($getters as $getter) {
-            if (method_exists($object, $getter)) {
+            if (method_exists($object, $getter) && is_callable([$object, $getter])) {
```

Comment:
Is `method_exists` still useful?

Reply:
Yes, `is_callable()` returns `true` for nonexistent methods when the class defines `__call()`.